addons/account/models/account_move.py

```diff
@@ -848,7 +848,7 @@ def _apply_cash_rounding(self, diff_balance, diff_amount_currency, cash_rounding
                     'account_id': biggest_tax_line.account_id.id,
                     'tax_repartition_line_id': biggest_tax_line.tax_repartition_line_id.id,
                     'tax_tag_ids': [(6, 0, biggest_tax_line.tax_tag_ids.ids)],
-                    'exclude_from_invoice_tab': True,
+                    'line_type': 'inv_tax_cash_rounding',
```

Comment:
not sure you need this because this line wasn't `is_rounding_line` at the first place

Reply:
It was, if you look a bit above you see it is updating the values in the dict a bit above (which is is_rounding_line). 
On as deux line_type différend parce que l'un est exclus de invoice tab et pas l'autre.